minari/dataset/_storages/parquet_storage.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+import pathlib
+
+import gymnasium as gym
+
+from minari.dataset._storages.arrow_storage import ArrowStorage
+
+
+_FIXEDLIST_SPACES = (gym.spaces.Box, gym.spaces.MultiDiscrete, gym.spaces.MultiBinary)
+
+
+class ParquetStorage(ArrowStorage):
+    FORMAT = "parquet"
+
+    def __init__(
+        self,
+        data_path: pathlib.Path,
+        observation_space: gym.Space,
+        action_space: gym.Space,
+    ):
+        super().__init__(data_path, observation_space, action_space)
```

Comment:
Very small, but I would put this to the arrow_storage file, so you don't need to copy _FIXEDLIST_SPACES

More importantly, did you managed to run the benchmark?